Add `ViewInterface::deepClone()` + Bump dev packages (#278)
yiisoft/view: UPGRADE.md

> to upgrade from version A to version C and there is version B between A and C, you need
> to following the instructions for both A and B.

## Upgrade from 11.x

> - Changed `ViewInterface`. You should implement it in your own implementations of interface.

Reviewer: ```suggestion
- `ViewInterface` was changed. You should adjust it in your own implementations.
```

  - Added `deepClone()` method. 

>   - Allow to pass `null` to `withBasePath()` and `withContext()` methods.

Reviewer: ```suggestion
  - Allowed to pass `null` to `withBasePath()` and `withContext()` methods.
```
